specviz/widgets/workspace.py

```diff
@@ -354,6 +354,17 @@ def compose_filter_string(reader):
 
         self.load_data(file_path, file_loader=loader_name_map[fmt])
 
+    def _add_and_plot_data(self, spectrum, name):
```

Comment:
There already is a method `force_plot` that does what this method does. Is there are reason to duplicate that functionality?

Reply:
This was a leftover from the pre-rebase work that I did. This function is slightly different but can be refactored in terms of `force_plot`.